File: Bug_Hunt_Practice/Question 2/solution.py
def find_max(arr):
    max_val = 0 
    
    for num in arr:
        if num > max_val:
            max_val = num
    return max_val

# --- Main execution ---
numbers = [-10, -5, -3, -8]
max_num = find_max(numbers)

print(f"Expected Output: -3")
print(f"Actual Output:   {max_num}")

File: Bug_Hunt_Practice/Question 2/solution.cpp
#include <iostream>
using namespace std;

int findMax(int arr[], int size) {
    int maxVal = 0; 
    
    for (int i = 0; i < size; i++) {
        if (arr[i] > maxVal) {
            maxVal = arr[i];
        }
    }
    return maxVal;
}

int main() {
    int numbers[] = {-10, -5, -3, -8};
    int size = 4;
    
    int max = findMax(numbers, size);
    cout << "Expected Output: -3" << endl;
    cout << "Actual Output:   " << max << endl;
    return 0;
}